zeus/queue/__init__.py

```diff
@@ -0,0 +1,73 @@
+
+import threading
+
+from zeus.utils.imports import import_string
+
+from .base import Task
+from .signals import task_postrun, task_prerun
+
+
+class Queue(object):
```

Comment:
I think of this like the Flask extension + the core "app" instance for the queue. You can imagine a world where we broke this out into its own lib, and most of this would stay, other than the init_app bits.